kDrive/UI/Controller/Create File/PlusButtonFloatingPanelViewController.swift

```diff
@@ -241,7 +248,9 @@ class PlusButtonFloatingPanelViewController: UITableViewController, FloatingPane
     // MARK: Actions
 
     private func importAction(_ mainTabViewController: MainTabViewController) {
-        let documentPicker = DriveImportDocumentPickerViewController(documentTypes: [UTI.data.identifier], in: .import)
+        guard let utType = UTI.data.asUTType else { return }
+        let documentPicker = DriveImportDocumentPickerViewController(forOpeningContentTypes: [utType])
```

Comment:
We could simply do this instead of adding a translation layer ?
```suggestion
        let documentPicker = DriveImportDocumentPickerViewController(forOpeningContentTypes: [UTType.data])
```